plugins/TestRunner/Commands/CodeCoverage.php

```diff
@@ -24,12 +19,14 @@ protected function configure()
     {
         $this->setName('tests:coverage');
         $this->setDescription('Run all phpunit tests and generate a combined code coverage');
-        $this->addOption('testsuite', null, InputOption::VALUE_REQUIRED, 'Run only a specific test suite, for instance UnitTests, IntegrationTests or SystemTests.');
-        $this->addArgument('group', InputArgument::OPTIONAL, 'Run only a specific test group. Separate multiple groups by comma, for instance core,plugins', '');
+        $this->addRequiredValueOption('testsuite', null, 'Run only a specific test suite, for instance UnitTests, IntegrationTests or SystemTests.');
+        $this->addOptionalArgument('group', 'Run only a specific test group. Separate multiple groups by comma, for instance core,plugins', '');
     }
 
-    protected function execute(InputInterface $input, OutputInterface $output)
+    protected function doExecute(): int
     {
+        $input = $this->getInput();
+        $output = $this->getOutput();
         $phpCovPath = trim(shell_exec('which phpcov'));
```

Comment:
If phpcov is not installed then the message `Deprecated - trim(): Passing null to parameter #1 ($string) of type string is deprecated` is shown in PHP8.2.2

```suggestion
        $phpCovPath = trim(shell_exec('which phpcov') ?? '');
```

Reply:
Good catch, Ben!